packages/ember-views/lib/compat/attrs-proxy.js

```diff
@@ -67,50 +37,11 @@ let AttrsProxyMixin = {
     val.update(value);
   },
 
-  _propagateAttrsToThis() {
-    let attrs = this.attrs;
-
-    for (let prop in attrs) {
-      if (prop !== 'attrs' && !this.constructor.__avoidPropagating[prop]) {
-        this.set(prop, this.getAttr(prop));
-      }
-    }
-  },
-
-  initializeShape: on('init', function() {
-    this._isDispatchingAttrs = false;
-  }),
-
-  _internalDidReceiveAttrs() {
-    this._super();
+  propagateAttrsToThis(attrs) {
```

Comment:
Is this intended to be public API?  Seems like it isn't, so we should keep it underscored.  If it is, we need docs.